version.sbt

```
@@ -1 +1 @@
-version in ThisBuild := "0.6.0-SNAPSHOT"
+version in ThisBuild := "0.7.0-SNAPSHOT"
```

```suggestion
version in ThisBuild := "0.6.0-SNAPSHOT"
```

We could deploy it as part of the current version :)